airtime_mvc/application/common/HTTPHelper.php

```diff
@@ -37,6 +37,9 @@ public static function getStationUrl($secured = true)
         if ($baseDir[0] != "/") {
             $baseDir = "/" . $baseDir;
         }
+        if ($baseDir[strlen($baseDir)-1] != "/") {
```

Comment:
You can use `substr($baseDir, -1)` to get the strings last char in a more straight forward fashion.

Reply:
Great, I updated the code.. I guess not but.. should I send the request again?